pkg/component/worker/kubelet.go

```diff
@@ -42,7 +42,7 @@ import (
 	kubeletv1beta1 "k8s.io/kubelet/config/v1beta1"
 	"k8s.io/utils/pointer"
 
-	"github.com/docker/libnetwork/resolvconf"
+	"github.com/moby/moby/libnetwork/resolvconf"
```

Comment:
Wondering if it's worth pulling in such a big dependency just for [resolvconf.Path()](https://pkg.go.dev/github.com/Moby/Moby@v23.0.1+incompatible/libnetwork/resolvconf#Path). Maybe we should inline this?

Reply:
yup, would make sense